### DISCUSSION TITLE
Is there `--copy-chunker-params` equivalent when init a repo?

### DISCUSSION THREAD

**Discussion by zxcat** [rustic-rs/rustic#909] [Q&A] [answered]
Even though the copy command can transfer snapshots between arbitrary repositories, deduplication between snapshots from the source and destination repository may not work. Restic solves this with `--copy-chunker-params` when init repo.

How to do this in rustic?

**Comment by aawsome** [comment, marked as answer]
Instead of pre-initializing the repos you want to copy snapshots into, just use `rustic copy --init` for the first run - this will set the correct chunker parameters.

Note that rustic currently refuses to copy to a repository with different chunker parameters - IMO the right way to implement this action would be to re-chunk the data contained in the snapshot (which is not yet implemented).

**Comment by nadir-ishiguro** [comment]
This took me forever to figure out, though it makes total sense in hindsight. 

I think a prominent reference in the documentation would be helpful, as looking for "--copy-chunker-params" only pointed to the comparison page that says we don't need it anymore with rustic. 

I was missing the `--init` part. 

Thank you for rustic. It's a pleasure to use.

**Reply by aawsome** [reply]
Thanks for the hint. I updated the comparison page.
Note: You can easily propose changes to the docu yourself by just pressing the "suggest an edit" button in the upper right corner. Docu is hard to get good and we appreciate any help here!

**Reply by nadir-ishiguro** [reply]
Thank you! I'll keep that in mind and will do so the next time I find something to improve.

**Reply by simonsan** [reply]
Related issue: https://github.com/rustic-rs/docs/issues/79

**Reply by simonsan** [reply]
Documented here now: https://rustic.cli.rs/docs/commands/misc/copy.html#copying-to-a-repository-with-different-chunker-parameters
